sql/parser/expr.go

```diff
@@ -247,6 +247,17 @@ func (n QualifiedName) Table() string {
 	return n[len(n)-1]
 }
 
+// Column returns the column portion of the name.
+//
+// TODO(pmattis) Handling of qualified names is currently very basic and we
+// consider the column portion of the name to simply be the last
+// component. This is identical to the table portion of the name. Perhaps it
+// would be better to name this the "last" component because context matters in
```

Comment:
if we had the types and methods `Name() string` and `Parent() <sometype>` that would flow nicely:

```go
cname.Name() // column name
cname.Parent().Name() // table name
dname := cname.Parent().Parent()
dname.Name() // database name
```